types/artifacts.d.ts

```diff
@@ -144,23 +144,31 @@ declare global {
       export type TaskNode = _TaskNode;
       export type MetaElement = LH.Artifacts['MetaElements'][0];
 
-      export interface Accessibility {
-        violations: {
-          id: string;
-          impact: string;
-          tags: string[];
-          nodes: {
-            path: string;
-            html: string;
-            snippet: string;
-            target: string[];
-            failureSummary?: string;
-            nodeLabel?: string;
-          }[];
-        }[];
-        notApplicable: {
-          id: string
+      export interface RuleExecutionError {
+        name: string;
+        message: string;
+      }
+
+      export interface AxeResult {
+        id: string;
+        impact: string;
+        tags: string[];
+        nodes: {
+          path: string;
+          html: string;
+          snippet: string;
+          target: string[];
+          failureSummary?: string;
+          nodeLabel?: string;
         }[];
+        error?: RuleExecutionError;
+      }
+
+      export interface Accessibility {
+        passes: AxeResult[];
```

Comment:
we don't seem to return a `passes` property?

Reply:
Good catch, thanks :)